.identity/.gitignore

```diff
@@ -0,0 +1,23 @@
+# Local .terraform directories
```

Comment:
I wouldn't create another .gitignore but instead I'd add tf things in the main file